packages/jellyfish-api-core/__tests__/category/loan/closeVault.test.ts

```diff
@@ -34,7 +35,10 @@ describe('Loan', () => {
     const collateralAddress = await tGroup.get(0).container.getNewAddress()
     await tGroup.get(0).token.dfi({ address: collateralAddress, amount: 200000 })
 
-    // oracle setup
+    const loanTokenMinterGroup = tGroup.get(2)
```

Comment:
already taking node no 3. so no need the `group` part. update the name?
```suggestion
    const loanTokenMinter = tGroup.get(2)
```